scripts/items/tarutaru_snare.lua

```diff
@@ -12,7 +12,12 @@ itemObject.onItemCheck = function(target, item, param, caster)
 end
 
 itemObject.onItemUse = function(target)
-    -- TODO: reduce damage
+    target:setMobMod(xi.mobMod.BASE_DAMAGE_MULTIPLIER, 1)
+    target:timer(60000, function(mobArg)
+        if mobArg and mobArg:isAlive() then
+            mobArg:setMobMod(xi.mobMod.BASE_DAMAGE_MULTIPLIER, 100)
+        end
+    end)
```

Comment:
I think, just to be extra sure, you should also edit the mob scripts and set their damage multipliers properly in onMobSpawn.